register/processUserForm.php

```diff
@@ -96,6 +96,14 @@
 
 	$NewUser->options->set($_REQUEST['userForm']);
 	$NewUser->options->load();
+
+	// Give user access to tutorial views for 30 days

```

Comment:
Why just 30 days? Should we have it till they end up checking out the tutorials?

Reply:
Thanks for adjusting this